test/parallel/test-cluster-worker-disconnect.js

```diff
@@ -98,9 +98,9 @@ if (cluster.isWorker) {
 
     // flags
     assert.strictEqual(w.state, 'disconnected',
-                       'The state property was not set');
+                       'The state property was not set, current state is ${w.state}');
     assert.strictEqual(w.voluntaryMode, true,
-                       'Voluntary exit mode was not set');
+                       'Voluntary exit mode was not set, current voluntary mode is ${w.voluntaryMode}');
```

Comment:
The line length is to long. It would actually be best to just remove the message overall. The message from `strictEqual` should be better than the current situation. The same applies to `w.state`.